WireUI/Sources/WireIndividualToTeamMigrationUI/Views/TeamNameView.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+import WireFoundation
+
+struct TeamNameView: View {
+
+    enum Action: Sendable {
+        case `continue`(teamName: String)
+    }
+
+    let actionCallback: (Action) -> Void
+    @State var teamName: String = ""
```

Comment:
I think we should aim for `private` state variables as suggested by [apple](https://developer.apple.com/documentation/swiftui/state#):

> Declare state as private to prevent setting it in a memberwise initializer, which can conflict with the storage management that SwiftUI provides